extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -418,6 +418,18 @@ export class AmpStoryPage extends AMP.BaseElement {
     return layout == Layout_Enum.CONTAINER;
   }
 
+  /**
+   * Return true if the current AmpStoryPage is protected by a paywall.
+   * @return {boolean}
+   */
+  isPaywallProtected() {
+    return (
+      this.element.getAttribute('subscriptions-section') ===
+        'limited-content' ||
+      this.element.getAttribute('subscriptions-section') === 'content'
```

Comment:
What's the difference between these two values? Is it even necessary to have both?

Reply:
One is the dialog paywall page, e.g. the first locked page, and the other one is for normal locked pages. Yes it would be necessary because while navigating to the next page, if it's a normal locked page the dialog should be triggered immediately, whereas if it's the first locked page, the dialog should be triggered after a short period of time.

Added comments to clarify that.

Reply:
If you don't care about the value, you could just check `this.element.hasAttribute("subscriptions-section")` and avoid checking for both types

Reply:
Done. But this function would need to separate when in the future PRs we need to know the page is a limited content or a content one.